### PR TITLE
Czech translations

### PR DESCRIPTION
**Pull request by MrJaroslavik**
Czech translation added. Can you please create folder for it and rename this to strings.xml

### CODE REVIEW

**Comment by flex3r** [comment]
Not sure why I should do that, but sure. Also, strings marked as `translatable="false"` should not be translated or added to the file.  
Looks good otherwise, I'll have some others look over the translations and get back to you 👍

**Comment by MrJaroslavik** [comment]
Oh sorry, this is why in other languages is only 145 lines :D. I thought it should be there, sorry for that. And question: Where is language change option or it will be in some upcoming update? Thanks.

**Comment by MrJaroslavik** [comment]
And why - i wasn't able to create folder.

**Comment by flex3r** [comment]
There's no language option in DankChat right now, the app will use what language is set in the OS.

File: app/src/main/res/values-cz/strings.xml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="preference_suggestions_title">Návrhy uživatelů a emotikon</string>
+    <string name="preference_suggestions_summary">Zobrazí návrhy emotikon a uživatelských jmen při psaní</string>
+    <string name="preference_load_message_history_title">Při zapnutí načíst historii zpráv</string>
+    <string name="preference_message_history_header">Historie zpráv</string>
+    <string name="preference_message_history_dashboard_title">Otevřít řídící panel</string>
+    <string name="preference_message_history_dashboard_summary">Zjistěte více o službě a deaktivujte historii zpráv pro svůj vlastní kanál</string>
+    <string name="preference_channel_data_header">Data kanálu</string>
+    <string name="preference_developer_header">Nastavení vývojáře</string>
+    <string name="preference_debug_mode_title">Režim ladění</string>
+    <string name="preference_debug_mode_summary">Poskytuje informace o všech výjimkách, které byly zachyceny</string>
```
Comment (fabZeef) [inline app/src/main/res/values-cz/strings.xml:124]: Poskytuje informace o všech chybách, které byly zachyceny

File: app/src/main/res/values-cz/strings.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="preference_timestamp_format_title">Formát časových razítek</string>
+    <string name="preference_tts_title">Povolit TTS</string>
+    <string name="preference_tts_summary">Čte zprávy aktivního kanálu</string>
+    <string name="preference_tts_queue_title">Režim fronty zpráv</string>
```
Comment (fabZeef) [inline app/src/main/res/values-cz/strings.xml:128]: Režim čtení bez přerušování

File: app/src/main/res/values-cz/strings.xml [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="preference_tts_queue_add">Hrát každou zprávu s použitím fronty</string>
```
Comment (fabZeef) [inline app/src/main/res/values-cz/strings.xml:129]: Čte každou zprávu s použitím fronty

File: app/src/main/res/values-cz/strings.xml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="preference_tts_queue_flush">Hrát pouze nejnovější zprávu</string>
```
Comment (fabZeef) [inline app/src/main/res/values-cz/strings.xml:130]: Čte pouze nejnovější zprávu

File: app/src/main/res/values-cz/strings.xml [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="preference_tts_message_format_message">Číst pouze zprávu</string>
+    <string name="preference_tts_message_format_combined">Číst uživatele a zprávu</string>
+    <string name="preference_tts_message_format_title">Formát režimu zpráv</string>
```
Comment (fabZeef) [inline app/src/main/res/values-cz/strings.xml:133]: Formát režimu čtení zpráv

File: app/src/main/res/values-cz/strings.xml [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+    <string name="preference_tts_header">TTS</string>
+    <string name="preference_checkered_lines_title">Kostkované Řádky</string>
```
Comment (fabZeef) [inline app/src/main/res/values-cz/strings.xml:135]: Kostkované řádky

**Review by fabZeef — CHANGES_REQUESTED** [review]
Fixed some errors in translate

**Comment by flex3r** [comment]
@MrJaroslavik any progress on this?

**Pull request merged**